framework/encryption/__init__.py

```diff
@@ -6,13 +6,22 @@
 
 SENSITIVE_DATA_KEY = jwe.kdf(settings.SENSITIVE_DATA_SECRET.encode('utf-8'), settings.SENSITIVE_DATA_SALT.encode('utf-8'))
 
+def ensure_str(value):
```

Comment:
@chrisseto adding  `-*- coding: utf-8 -*-`seemed to be required for python not to get mad on parsing the tests. 🤷‍♂️